### PR TITLE
Add arm32 to ci

### PR DESCRIPTION
**Pull request by Oscar-Pepper**

### CODE REVIEW

File: scripts/integration_tests.sh [hunk 1]
```diff
@@ -41,29 +42,29 @@ function wait_for() {
...
-while getopts 'a:sl:t:h' OPTION; do
+while getopts 'a:l:t:sx:h' OPTION; do
     case "$OPTION" in
         a)
             abi="$OPTARG"
             case "$abi" in
                 x86_64)
                     api_lvl_default="30"
-                    api_tgt_default="google_apis"
+                    api_tgt_default="google_apis_playstore"
```
Comment (fluidvanadium) [inline scripts/integration_tests.sh]: i think the playstore version is a little bigger, but what advantage does it provide?

**Review by fluidvanadium — COMMENTED** [review]
i dont see any changes that have to do with arm32

File: scripts/integration_tests.sh (hunk 1 above)
Comment (Oscar-Pepper) [reply scripts/integration_tests.sh]: yes this is a good point it is bigger but the playstore image was the only one (other than google-tv hehe) that i could get arm APKs to not crash on.
https://github.com/zingolabs/zingo-mobile/pull/286/files#diff-ae9d5b5a1138bcf0c79a55f58cc65a7428fb14580f7b69c960d186e0ea1d24eeR139-R140

Comment (fluidvanadium) [reply scripts/integration_tests.sh]: huh. what a strange solution

**Comment by Oscar-Pepper** [comment]
> i dont see any changes that have to do with arm32

the key changes to do with adding arm32 is the addition to CI matrix:
https://github.com/zingolabs/zingo-mobile/pull/286/files#diff-944291df2c9c06359d37cc8833d182d705c9e8c3108e7cfe132d61a06e9133ddR198
and also the changes to get it working, the key line being:
https://github.com/zingolabs/zingo-mobile/pull/286/files#diff-ae9d5b5a1138bcf0c79a55f58cc65a7428fb14580f7b69c960d186e0ea1d24eeR67

**Review by fluidvanadium — DISMISSED** [review]

**Review by juanky201271 — APPROVED** [review]
looks good.

**Pull request merged**